gtfu/src/web/graas-report-web.js

```diff
@@ -97,26 +118,57 @@ function handleAgencyChoice(){
 
 function handleDateChoice(){
     console.log("handleDateChoice()");
+    isNewPageSelect = true;
     selectedDate = document.getElementById("date-select").value;
-    url = `${bucketURL}/graas-report-archive/${selectedAgency}/${selectedAgency}-${selectedDate}.json`;
+    // "?nocache=" mostly forces the page to reload files each time
+    url = `${bucketURL}/graas-report-archive/${selectedAgency}/${selectedAgency}-${selectedDate}.json?nocache=${(new Date()).getTime()}`;
     loadJSON(url, processTripJSON);
 }
 
 function processTripJSON(object){
     trips.length = 0;
-    for (var i = 0; i < object.trips.length; i++) {
-        var map = {x: object["trips"][i]["boundaries"]["map-x"],
```

Comment:
It would be easier to read as `object.trips[i].boundaries.mapx`, which of course would require removing all dashes from field names or replacing them with `_`.

Reply:
Done! Not sure why github isn't showing this as "outdated".

Reply:
Nice!